<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cadastroQuartos</class>
 <widget class="QDialog" name="cadastroQuartos">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>635</width>
    <height>458</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cadastro de quartos</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>234</width>
     <height>28</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Valor da diária: R$</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="txt_valor"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>23</x>
     <y>47</y>
     <width>581</width>
     <height>248</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Titulo do quarto:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txt_tituloQuarto"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Descrição do quarto:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="txt_descricao"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>641</width>
     <height>90</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Adicionais:</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QCheckBox" name="check_wifi">
          <property name="text">
           <string>Wi-Fi</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="check_ar">
          <property name="text">
           <string>Ar condicionado</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="check_tv">
          <property name="text">
           <string>TV</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="check_sacada">
          <property name="text">
           <string>Sacada</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="check_cofre">
          <property name="text">
           <string>Cofre</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="check_banheira">
          <property name="text">
           <string>Banheira</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="check_cafe">
          <property name="text">
           <string>Café da manhã</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QCheckBox" name="check_roupao">
          <property name="text">
           <string>Roupão e chinelos</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="check_higiene">
          <property name="text">
           <string>Produtos de higiene</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="check_servico">
          <property name="text">
           <string>Serviço de quarto</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="check_miniBar">
          <property name="text">
           <string>Mini bar</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QComboBox" name="boxTipoQuarto">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>20</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Simples</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Comfort</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Suíte</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Master</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>24</x>
     <y>21</y>
     <width>80</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Tipo de quarto:</string>
   </property>
  </widget>
  <widget class="QToolButton" name="btn_imagen">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>20</y>
     <width>22</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>430</y>
     <width>195</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="btn_salvar">
      <property name="text">
       <string>Salvar</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_cancelar">
      <property name="text">
       <string>Cancelar</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
